content/general/concepts/full-stack/full-stack.md

```diff
@@ -0,0 +1,78 @@
+---
+Title: 'Full-Stack'
+Description: 'Full-stack development refers to having competency in both client & server side software.'
+Subjects:
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Browsers'
+  - 'Debugging'
+  - 'Development'
+  - 'Frameworks'
+  - 'Responsive'
+  - 'Servers'
+CatalogContent:
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/back-end-engineer-career-path'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+Full-stack development refers to having competency in both client & server side software. In other words, full-stack developers have the skills needed to build entire applications from the [back-end](https://www.codecademy.com/resources/docs/general/back-end), through [APIs](https://www.codecademy.com/resources/docs/general/api), to the [front-end](https://www.codecademy.com/resources/docs/general/front-end). Full-stack developers are able to handle both front-end and back-end tasks. Typically, full-stack developers are stronger in either front or back-end and throughout their careers, gain competence in the full-stack.
+
+## Hiring Trends
+
+Startups tend to seek full-stack developers for several reasons, one of which is funding. Larger companies have the resources to hire large development teams, which allows them to have teams where developers can specialize in either front-end or back-end. Startups on the other hand, may be limited in their ability to hire a large development team, so instead will seek full-stack developers.
+
+## Common Tasks
+
+Full-stack web development can involve a variety of front-end and back-end tasks, including:
+
+- Creating and updating webpages.
+- Making sure webpages are responsive and work well on phones or tablets.
+- Improving website performance.
+- Adding and improving accessibility features.
+- Using front-end frameworks to build client-side software.
+- Building and consuming [APIs](https://www.codecademy.com/resources/docs/general/api).
+- Creating, integrating, and managing databases.
+- Using back-end frameworks to build server-side software.
+- Validating data to make sure it's formatted correctly before being sent to the database.
+- Integrating user-facing elements with server-side elements to make sure that information is being sent to the right place so the server can retrieve it.
+
+## Popular Full-Stack Tech Stacks
+
+Full-stack developers typically utilize a combination of software, called a stack, in order to build full-stack applications.
+
+### MERN Stack
+
+A couple highlights for the MERN stack include the full-stack JavaScript nature and the fact that React is used, which is popular with both developers and designers.
+
+- **M**[ongoDB](https://www.codecademy.com/learn/learn-mongodb): NoSQL database that uses [JSON](https://www.codecademy.com/resources/docs/general/json) (BSON)
```

Comment:
While I see what you're doing with these lists, I think it will be preferable to have the links be the whole word, and either drop the bolding of the initial letter or bold the whole word.